src/main/gov/nasa/jpf/jdart/JDart.java

```diff
@@ -257,6 +257,7 @@ public String format(LogRecord record) {
         
         //logger.info("Initial valuation: ", ca.getInitialValuation());
         if (!config.getBoolean("jdart.tree.dont.print")) {
+          ca.getConstraintsTree().toJson();
```

Comment:
You should add a different option for outputting to json, e.g. `jdart.tree.print.json`